src/core/plugins/configs/actions.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @prettier
+ */
+import { parseYamlConfig } from "./helpers"
```

Comment:
Please rename helpers to `fn.js` and rename `parseYamlConfig` to just `parseConfig` as this function is format agnostic.